README.md

```diff
@@ -11,13 +11,16 @@ $ cd my-project
 $ npm i --save-dev carbon-components-angular carbon-components @ngx-translate/core
 ```
 
-Then we need to include carbon-components in `src/styles.scss`:
+Then we need to include carbon-components in `src/styles.scss` and set our fonts path:
 
 ```scss
+$font-path: '/assets/fonts/';
 @import "~carbon-components/scss/globals/scss/styles.scss";
 ```
 
-And set up our translations in `src/app/app.module.ts`:
+And copy the fonts files from `node_modules/carbon-components/src/globals/fonts` into our app's `assets/fonts` folder.
```

Comment:
There should be a better way thank copying directly from node_modules?

Reply:
That's not a bad way ... It should be consistent, but I'm wondering if adding a "note: for offline usage use `@ibm/plex` and load fonts from there" or something would be best?

Reply:
Would `~carbon-components/src/globals/fonts` work?

Reply:
Yeah I think the default/encouraged use case should be to use a CDN, and this text should look something like:

> **note:** For offline usage we'll need to copy the fonts from `node_modules/carbon-components/src/globals/fonts` into our app's `assets/fonts` folder. and set `$font-path: '/assets/fonts/';` before our carbon components import.